docker/imposter/agents/agents.json

```diff
@@ -32,8 +32,12 @@
         "manager": "wazuh-manager-master-0",
         "node_name": "master",
         "lastKeepAlive": "9999-12-31T23:59:59Z",
+        "version": "Wazuh v4.5.0",
+        "group_config_status": "not synced",
+        "status_code": 0
```

Comment:
The property `status_code` is duplicated

Reply:
Solved in https://github.com/wazuh/wazuh-kibana-app/pull/5680/commits/18a0e95ff5bdaad7b44021be745ca77c701f0bea.